scheme/common/cca/realm/cca_realm_extractor.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package cca
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/services/handler"
+)
+
+type CcaRealmExtractor struct {
+	Scheme    string
+	SubScheme string
```

Comment:
`SubScheme` not necessary?

Reply:
Has been addressed in the later commits, the `SubScheme` was removed subsequently!